Source/VolunteerReporting/Web/Models/AutomaticReplyTypeModel.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Web.Models
+{
+    public class AutomaticReplyTypeModel
```

Comment:
Since it sits in Web.Models - does it need to suffixed by Model again?

Reply:
Working with both the enum which actually a type and a representation for a view model, this adds something... If we rather use full namespace, or some *using xx as name*, we can skip this. Let me know what you think!

Reply:
Not a biggy, I just don't like to have technical concerns added as prefix or suffix to types - the name should clearly say what it is, not where it fits in technically. Is the enum named exactly the same? If so, maybe there is the need to look at naming one of these differently.. ??

Reply:
I agree. The enum is actually named DefaultAutomaticReplyType, but I found the Default-prefix here should be removed too. (Types are not specific to the defaults). Would it be better to use a name that more describes the use, such as AutomaticReplyTypeName?

Reply:
Sure.. Sounds better.